extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -47,6 +50,40 @@ export async function findLanguage(
   return language;
 }
 
+interface RepoListQuickPickItem extends QuickPickItem {
+  repoList: string[];
+}
+
+/**
+ * Gets the repositories to run the query against.
+ */
+async function getRepositories(): Promise<string[] | undefined> {
+  if (REPO_LISTS) {
+    const quickPickItems = Object.keys(REPO_LISTS).map<RepoListQuickPickItem>(key => (
+      {
+        label: key,                // the name of the repository list
+        repoList: REPO_LISTS[key], // the actual array of repositories
+      }
+    ));
+    const quickpick = await window.showQuickPick<RepoListQuickPickItem>(
+      quickPickItems,
+      {
+        placeHolder: 'Select a repository list. You can define repository lists in the `codeQL.remoteRepositoryLists` setting.',
+        ignoreFocusOut: true,
+      });
+    if (quickpick && quickpick.repoList.length > 0) {
```

Comment:
nit:

```suggestion
    if (quickpick?.repoList.length > 0) {
```

Reply:
I tried this and got an error in the lines below that "Object `quickpick` might be undefined" 😅

Reply:
Hmmm...ok. Not sure what's happening, but fine to ignore.